### PR TITLE
LMS-005: Adding migrarion,seeder and factory for authorm,category,publisher

### PR DESCRIPTION
**Pull request by Mitali-Gautam**
This branch is about adding migrations and seeders for authors,publishers and categories

Command : 
php artisan migrate
php artisan db:seed

### CODE REVIEW

File: database/factories/AuthorFactory.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            "name"=>$this->faker->name,
+            "created_at"=>$this->faker->dateTime(),
+            "updated_at"=>$this->faker->dateTime()
```
Comment (mehulprajapatii) [inline database/factories/AuthorFactory.php:22]: Use PSR 4 ,
            'updated_at' => $this->faker->dateTime(),

File: database/factories/AuthorFactory.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            "name"=>$this->faker->name,
```
Comment (mehulprajapatii) [inline database/factories/AuthorFactory.php:20]: Use single quote

File: database/factories/CategoryFactory.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            "name"=>$this->faker->name,
```
Comment (mehulprajapatii) [inline database/factories/CategoryFactory.php:20]: use single quote and PSR 4 coding standard

File: database/factories/StudentFactory.php [hunk 4]
```diff
@@ -23,7 +23,9 @@ public function definition(): array
                 "email"=>$this->faker->safeEmail,
                 "phone"=>$this->faker->phoneNumber,
                 "address"=>$this->faker->address,
-                "class"=>$this->faker->text($maxNbChars = 10)
+                "class"=>$this->faker->text($maxNbChars = 10),
+                "created_at"=>$this->faker->dateTime(),
+                "updated_at"=>$this->faker->dateTime()
```
Comment (mehulprajapatii) [inline database/factories/StudentFactory.php:28]: Add , after last array element

File: database/migrations/2023_08_19_075102_create_publishers_table.php [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('publishers', function (Blueprint $table) {
+            $table->id()->autoIncrement();
+            $table->string("name",100);
```
Comment (mehulprajapatii) [inline database/migrations/2023_08_19_075102_create_publishers_table.php:16]: space after comma e.g. 'name', 100

File: database/migrations/2023_08_19_075102_create_publishers_table.php [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+            $table->enum("is_deleted",[1,0])->default(0);
```
Comment (mehulprajapatii) [inline database/migrations/2023_08_19_075102_create_publishers_table.php:17]: Add ->comment('usage of column'); for description of column

File: database/seeders/DatabaseSeeder.php [hunk 7]
```diff
@@ -5,22 +5,19 @@
...
 {
     /**
      * Seed the application's database.
      */
     public function run(): void
     {
-        Student::factory(10)->create();
-
-        // \App\Models\User::factory()->create([
-        //     'name' => 'Test User',
```
Comment (mehulprajapatii) [inline database/seeders/DatabaseSeeder.php:19]: remove commented code

**Review by mehulprajapatii — CHANGES_REQUESTED** [review]

**Review by mehulprajapatii — APPROVED** [review]

File: database/factories/AuthorFactory.php (hunk 2 above)
Comment (Mitali-Gautam) [reply database/factories/AuthorFactory.php]: Done

File: database/factories/StudentFactory.php (hunk 4 above)
Comment (Mitali-Gautam) [reply database/factories/StudentFactory.php]: Done

File: database/migrations/2023_08_19_075102_create_publishers_table.php (hunk 5 above)
Comment (Mitali-Gautam) [reply database/migrations/2023_08_19_075102_create_publishers_table.php]: Done

File: database/seeders/DatabaseSeeder.php (hunk 7 above)
Comment (Mitali-Gautam) [reply database/seeders/DatabaseSeeder.php:19]: Done

**Review by mehulprajapatii — APPROVED** [review]

**Pull request merged**